### PR TITLE
Cherry Picking Indigo release ready changes into Kinetic

### PR DESCRIPTION
**Pull request by jrgnicho**
This PR does the following:
- Cherry-picked all the changes from the indigo branch made prior to its release
- Updated the .travis.yaml file
- @Levi-Armstrong after these changes can we make kinetic-devel the main branch?

### CODE REVIEW

**Comment by Levi-Armstrong** [comment]
>@Levi-Armstrong after these changes can we make kinetic-devel the main branch?

Yea that is fine with me.

**Comment by jrgnicho** [comment]
@Levi-Armstrong the build passed

File: constrained_ik/CHANGELOG.rst [hunk 1]
```diff
@@ -0,0 +1,11 @@
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Changelog for package constrained_ik
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+0.1.1 (2017-03-24)
+------------------
+
```
Comment (Levi-Armstrong) [inline constrained_ik/CHANGELOG.rst:7]: I think if there were no changes we either add *No Changes* or something along those lines.

File: constrained_ik/constrained_ik_constraint_plugin_description.xml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-<library path="lib/libconstrained_ik_constraints">
+<library path="libconstrained_ik_constraints">
```
Comment (Levi-Armstrong) [inline constrained_ik/constrained_ik_constraint_plugin_description.xml:1]: Just wanted to check, is this a new change in Kinetic or has this always been wrong?

File: industrial_collision_detection/CHANGELOG.rst [hunk 3]
```diff
@@ -0,0 +1,11 @@
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Changelog for package industrial_collision_detection
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+0.1.1 (2017-03-24)
+------------------
+
```
Comment (Levi-Armstrong) [inline industrial_collision_detection/CHANGELOG.rst:7]: Same as above.

File: industrial_moveit/CHANGELOG.rst [hunk 4]
```diff
@@ -0,0 +1,11 @@
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Changelog for package industrial_moveit
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+0.1.1 (2017-03-24)
+------------------
+
```
Comment (Levi-Armstrong) [inline industrial_moveit/CHANGELOG.rst:7]: Same as above

File: constrained_ik/constrained_ik_constraint_plugin_description.xml (hunk 2 above)
Comment (jrgnicho) [reply constrained_ik/constrained_ik_constraint_plugin_description.xml:1]: I changed this while trying to troubleshoot something, I didn't restore it back to how it was but it seems that catkin is smart enough to figure out where to find the lib.

File: constrained_ik/CHANGELOG.rst (hunk 1 above)
Comment (jrgnicho) [reply constrained_ik/CHANGELOG.rst:7]: This came from git-cherry-picking, I can update the change logs once I try to get it ready for release.

**Pull request merged**